### PR TITLE
Moved SliderGroup to FC.

### PR DESCRIPTION
**Pull request by NitzanWix**
## Description
Converted SliderGroup to FC.

## Changelog
SliderGroup - Moved to a FC.

### CODE REVIEW

**Comment by Inbal-Tish** [comment]
@NitzanWix Please add  the "changelog" filed on all PRs

File: src/components/slider/context/SliderGroup.tsx [hunk 1]
```diff
@@ -11,40 +12,24 @@ interface SliderGroupProps {
...
-
-export default class SliderGroup extends Component<SliderGroupProps, SliderGroupState> {
-  static displayName = 'IGNORE';
+const SliderGroup = (props: SliderGroupProps) => {
+  const {color, onValueChange, children} = props;
+  const [value, setValue] = useState(Colors.getHSL(color));
 
-  constructor(props: SliderGroupProps) {
-    super(props);
+  const valueSetter = useCallback((value: tinycolor.ColorFormats.HSLA) => {
```
Comment (Inbal-Tish) [inline src/components/slider/context/SliderGroup.tsx:19]: 'valueSetter' is not a proper name for a callback. We usually use the name of the prop it is passed to or as an action describing what this method does

File: src/components/slider/context/SliderGroup.tsx [hunk 2]
```diff
@@ -11,40 +12,24 @@ interface SliderGroupProps {
...
+    setValue(value);
+    onValueChange?.(Colors.getHexString(value));
+  },
+  [onValueChange]);
 
-    this.state = {
-      value: Colors.getHSL(props.color)
-    };
-  }
+  const providerValue = useMemo(() => ({value, setValue: valueSetter}), [value, valueSetter]);
```
Comment (Inbal-Tish) [inline src/components/slider/context/SliderGroup.tsx:25]: Naming - 'contextProviderValue' - use a more descriptive name as "provider" is not clear enough

File: src/components/slider/context/SliderGroup.tsx (hunk 1 above)
Comment (NitzanWix) [reply src/components/slider/context/SliderGroup.tsx]: Is _setValue better?

**Review by Inbal-Tish — APPROVED** [review]

File: src/components/slider/context/SliderGroup.tsx (hunk 1 above)
Comment (Inbal-Tish) [reply src/components/slider/context/SliderGroup.tsx]: Yes

**Pull request merged**